pytorch_transformers/tokenization_bert.py

```diff
@@ -166,6 +166,14 @@ def convert_tokens_to_string(self, tokens):
         out_string = ' '.join(tokens).replace(' ##', '').strip()
         return out_string
 
+    def add_special_tokens_single_sentence(self, token_ids):
+        return [self._convert_token_to_id(self.cls_token)] + token_ids + [self._convert_token_to_id(self.sep_token)]
+
+    def add_special_tokens_sentences_pair(self, *token_ids):
```

Comment:
I like that! Should we make the function more specifically targeted to sentence pairs with a signature `def add_special_tokens_sentences_pair(self, token_ids_0, token_ids_1)` to avoid misunderstanding?
Also we need a doctring here!